### PR TITLE
Chart docs: note uid write permissions for existing pvc

### PR DESCRIPTION
**Pull request by jedcunningham**
Simple note to clarify the Airflow user needs write permission on existing pvc's.

### CODE REVIEW

**Review by kaxil — APPROVED** [review]

File: docs/helm-chart/manage-logs.rst [hunk 1]
```diff
@@ -75,6 +75,8 @@ In this approach, Airflow will log to an existing ``ReadWriteMany`` PVC. You pas
       --set logs.persistence.enabled=true \
       --set logs.persistence.existingClaim=my-volume-claim
 
+Note that the Airflow user (default uid 50000) needs write permission on the volume.
```
Comment (kaxil) [inline docs/helm-chart/manage-logs.rst]:
```suggestion
Note that the Airflow user (default uid ``50000``) needs write permission on the volume.
```

**Comment by kaxil** [comment]
Tested this locally -- works fine.

**Comment by potiuk** [comment]
Technically  speaking you need 'root' (0) group write access @jedcunningham @kaxil

**Comment by potiuk** [comment]
You can run airflow images with any user as long as you run it with group = 0

**Comment by potiuk** [comment]
See https://airflow.apache.org/docs/docker-stack/entrypoint.html#allowing-arbitrary-user-to-run-the-container

**Comment by potiuk** [comment]
Such arbitrary user has to be able to write to certain directories that needs write access, and since it is not advised to allow write access to “other” for security reasons, the OpenShift guidelines introduced the concept of making all such folders have the 0 (root) group id (GID). All the directories that need write access in the Airflow production image have GID set to 0 (and they are writable for the group). We are following that concept and all the directories that need write access follow that.

The GID=0 is set as default for the airflow user, so any directories it creates have GID set to 0 by default. The entrypoint sets umask to be 0002 - this means that any directories created by the user have also “group write” access for group 0 - they will be writable by other users with root group. Also whenever any “arbitrary” user creates a folder (for example in a mounted volume), that folder will have a “group write” access and GID=0, so that execution with another, arbitrary user will still continue to work, even if such directory is mounted by another arbitrary user later.

**Comment by jedcunningham** [comment]
Thanks @potiuk, I somehow never quite connected the dots previously. I've opened #17177 to fix it up.

**Pull request merged**